python/py-Pillow/Portfile

```diff
@@ -19,9 +19,9 @@ long_description    ${description}
 
 homepage            https://github.com/python-imaging/Pillow
 
-checksums           rmd160  c20a0bab9a469c3557044793174dd38905ce4be9 \
-                    sha256  b8e2f83c56e141920c39464b852de3719dfbfb6e3c99a2d8da0edf4fb33176ed \
-                    size    49368411
+checksums           rmd160  6fc017168f32f83acf1a561d23a2b643874caaef \
+                    sha256  ee6e2963e92762923956fe5d3479b1fdc3b76c83f290aad131a2f98c3df0593e \
+                    size    49513779
 
 if {${name} ne ${subport}} {
     conflicts           py${python.version}-pil
```

Comment:
I know this isn't  you change, but it would be good to fix the lint warning. There is only a py2{6,7} subport for `py-pil` so this conflict statements should become:
```
if {${python.version} == 27} { 
    conflicts       py${python.version}-pil
}
```